### PR TITLE
Fix travis

### PR DESCRIPTION
**Pull request by unnonouno**
Fix travis configuration file to test model branch.

### CODE REVIEW

**Comment by unnonouno** [comment]
Note that this patch doesn't pass travis CI as `model` branch includes bugs.

File: setup.py [hunk 1]
```diff
@@ -48,6 +48,7 @@
                       'numpy>=1.9.0',
                       'protobuf',
                       'six>=1.9.0'],
+    setup_requires=['Cython>=0.17'],
```
Comment (beam2d) [inline setup.py]: Would you add a short comment on why Cython is required here? (required for h5py?)

**Comment by beam2d** [comment]
Added a comment.

**Comment by beam2d** [comment]
LGTM!

**Comment by beam2d** [comment]
Thanks!

**Pull request merged**